lib/widgets/action_sheet.dart

```diff
@@ -36,20 +36,46 @@ void showMessageActionSheet({required BuildContext context, required Message mes
       && reactionWithVotes.userIds.contains(store.selfUserId))
     ?? false;
 
-  showDraggableScrollableModalBottomSheet<void>(
+  final designVariables = DesignVariables.of(context);
+  showModalBottomSheet<void>(
     context: context,
+    clipBehavior: Clip.antiAlias,
+    backgroundColor: designVariables.actionSheetBackground,
+    shape: const RoundedRectangleBorder(borderRadius: BorderRadius.vertical(top: Radius.circular(20.0))),
+    useSafeArea: true,
+    isScrollControlled: true,
     builder: (BuildContext _) {
-      return Column(children: [
-        if (!hasThumbsUpReactionVote) AddThumbsUpButton(message: message, messageListContext: context),
-        StarButton(message: message, messageListContext: context),
-        if (isComposeBoxOffered) QuoteAndReplyButton(
-          message: message,
-          messageListContext: context,
+      return Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            // TODO(#217): show message text
+            Flexible(
+              child: SingleChildScrollView(
+                child: ClipRRect(
+                  borderRadius: BorderRadius.circular(7),
+                  child: Column(children: [
+                    if (!hasThumbsUpReactionVote) AddThumbsUpButton(message: message, messageListContext: context),
+                    StarButton(message: message, messageListContext: context),
+                    if (isComposeBoxOffered) QuoteAndReplyButton(
+                      message: message,
+                      messageListContext: context,
+                    ),
+                    CopyMessageTextButton(message: message, messageListContext: context),
+                    CopyMessageLinkButton(message: message, messageListContext: context),
+                    ShareButton(message: message, messageListContext: context),
```

Comment:
The order doesn't match the Figma design, it should be:

```dart
AddThumbsUpButton(),
CopyMessageTextButton(),
CopyMessageLinkButton(),
ShareButton(),
StarButton(),
QuoteAndReplyButton(),
```

Reply:
Yeah, it is different. But [Alya decided](https://chat.zulip.org/#narrow/stream/48-mobile/topic/bottom.20sheet.20redesign.20-.20.23F853/near/1908715) to keep it as it is. 🙂